### ISSUE TITLE
SKipper takes the entire screen on Linux, so you can't see the Task Bar

### ISSUE THREAD

**Issue by ba58smith** [JohnySeven/SKipper#6]
This will be confusing, I think.

You can use ALT+TAB to get to another program, and then you can see the Task Bar, but I feel like the Task Bar should always be available, or there should be a way to change from Full Screen to "Normal Screen" mode. Maybe not, though, since SKipper is likely to be the only program running on the computer? I don't know - we should discuss.

**Comment by JohnySeven** [comment]
Yeah, let's discuss it! But it's related to desktop, but I can image having some kind of fullscreen x windowed mode that could user just toggle and Skipper would remember last state.

**Comment by ba58smith** [comment]
What about this? If you click or touch anywhere on the SKipper screen, it shows the Task Bar for 10 seconds (or whatever duration we decide makes sense). After that time, the Task Bar hides, and you're back to full screen. Signal K's Instrument Panel does something like that, to hide some controls that you don't need very often.

Since SKipper is primarily displaying data, users won't be touching / clicking on it during normal usage, I don't think. So this could work?
